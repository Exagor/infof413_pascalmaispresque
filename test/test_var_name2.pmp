begin
    whileif := 0
end