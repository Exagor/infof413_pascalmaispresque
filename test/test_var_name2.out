token: begin	lexical unit: BEG
token: whileif	lexical unit: VARNAME
token: :=	lexical unit: ASSIGN
token: 0	lexical unit: NUMBER
token: end	lexical unit: END
token: lol	lexical unit: VARNAME
Variables :
lol   5
whileif   2
